plugins/inputs/dmcache/dmcache.go

```diff
@@ -0,0 +1,49 @@
+package dmcache
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type DMCache struct {
+	PerDevice bool `toml:"per_device"`
+	rawStatus func() ([]string, error)
```

Comment:
I feel like the name `rawStatus` is not fitting, since this functions both executes the command and does some processing on the output.